src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.figure;
+
+public class IsoscelesTrapezoid extends AbstractFigure implements Figure {
+
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * height * ((base1 + base2) / 2);
```

Comment:
The formula for the area of an isosceles trapezoid seems to be incorrect. The correct formula is 0.5 * (base1 + base2) * height. Please correct the formula.